index.md

```diff
@@ -86,5 +86,5 @@ _**Legend:** A = Accepted | D = Draft | R = Review | X = Deprecated_
 [psr13]: /psr/psr-13/
 [psr14]: https://github.com/php-fig/fig-standards/blob/master/proposed/event-manager.md
 [psr15]: https://github.com/php-fig/fig-standards/blob/master/proposed/http-middleware
-[psr16]: https://github.com/php-fig/fig-standards/blob/master/proposed/simplecache.md
+[psr16]: /psr/psr-16/
```

Comment:
we have not yet updated the website, so this link will 404. Can you make it the github URL for now until the website is updated?